tasks/main.yml

```diff
@@ -13,6 +13,10 @@
     with_fileglob: "pre_checks/*.yml"
     when: he_pre_checks
 
+  - name: System configuration validations
```

Comment:
The same,
Move it to the end of the block or open another block - it's not belong to the main process.

An outsider who reads the code will not understand the idea of having the checks triggered again like this.